Makefile

```diff
@@ -22,6 +22,12 @@ DEV_IMG ?= # <== NOTE:  outside dev, change this!!!
 VERSION ?= $(shell git describe --exact-match 2> /dev/null || \
 	   git describe --match=$(git rev-parse --short=8 HEAD) --always --dirty --abbrev=8)
 
+KUSTOMIZE_VERSION = $(kustomize version | awk '{ print $2 }' | awk -F ':' '{ print $2} ')
+ifneq (,$(filter $(KUSTOMIZE_VERSION), "v1", "unknown"))
```

Comment:
Does filter check if it's v1 or unknown?  What will this line do if someone builds kustomize from source and the version doesn't have a correct version?

Reply:
True.  Now that I think about it, this is the behavior we want.  This code is fine.